docs/en/advanced/Alluxio-Storage-Management.md

```diff
@@ -355,3 +444,34 @@ $ bin/alluxio runTests -Dalluxio.user.file.create.ttl=1m -Dalluxio.user.file.cre
 
 Note, if you try this example, make sure the `alluxio.master.ttl.checker.interval` is set to a short
 duration, ie. 1 minute.
+
+<!-- ################## -->
+<!-- TODO(Zac): Add examples on check capacity, fraction of used capacity and etc -->
+<!-- ################## -->
+
+## Verifying Alluxio Cache Capacity and Usage
+
+The alluxio shell exposes two commands which allow a user to verify how much space is available to
+the Alluxio cache as well as how much space is currently in use. The `fs getCapacityBytes` and
+`fs getUsedBytes` commands show a user how many bytes of available and used storage are in the
+Alluxio cache respectively.
+
+To get the total used bytes:
+
+```bash
+./bin/alluxio fs getUsedBytes
+```
+
+To get the total capacity in bytes:
+
+```bash
+./bin/alluxio fs getCapacityBytes
+```
+
```

Comment:
I think `bin/alluxio fsadmin report` is more useful than `fs getUsedBytes/getCapacityBytes`. It gives output like
```
➜  alluxio-1.8.0 bin/alluxio fsadmin report
Alluxio cluster summary:
    Master Address: localhost/127.0.0.1:19998
    Web Port: 19999
    Rpc Port: 19998
    Started: 09-28-2018 12:52:09:486
    Uptime: 0 day(s), 0 hour(s), 0 minute(s), and 26 second(s)
    Version: 1.8.0
    Safe Mode: true
    Zookeeper Enabled: false
    Live Workers: 1
    Lost Workers: 0
    Total Capacity: 10.67GB
        Tier: MEM  Size: 10.67GB
    Used Capacity: 0B
        Tier: MEM  Size: 0B
    Free Capacity: 10.67GB
```

Reply:
Well that does seem quite a bit more useful. I'll add that in too.